src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -94,7 +104,7 @@ <h5 class="pb-1" jhiTranslate="artemisApp.examManagement.sections.conduction">Ex
                 <div class="col-sm-6">
                     <label class="form-check-label" for="workingTimeInMinutes" jhiTranslate="artemisApp.examManagement.workingTime">Working Time</label>
                     <jhi-help-icon [text]="'artemisApp.examManagement' + (exam.testExam ? '.testExam' : '') + '.workingTimeTooltip'"></jhi-help-icon>
-                    <input *ngIf="!exam.testExam" readonly type="text" class="form-control" [value]="calculateWorkingTime" />
```

Comment:
Note: Data flow was wild before, `calculateWorkingTime` was a getter that actually changed other data, the tests freaked out w/ this.